externs/ima.js

```diff
@@ -227,11 +227,18 @@ google.ima.AdEvent.Type = {
 
 /**
  * @typedef {{
+ *   adsResponse: ?string,
  *   adTagUrl: string,
  * }}
  *
  * @description Request for the ad server
  * @property {string} adTagUrl
+ *   Specifies the ad tag url that is requested from the ad server.
```

Comment:
You should include that this is optional if `adsResponse` is given.

Reply:
Done